core/src/main/java/org/apache/brooklyn/core/entity/EntityPredicates.java

```diff
@@ -488,11 +482,6 @@ public String toString() {
         }
     }
 
-    /** @deprecated since 0.7.0 use {@link #isManaged()} */ @Deprecated
-    public static <T> Predicate<Entity> managed() {
-        return isManaged();
-    }
-
     /** @deprecated since 0.7.0 use {@link #isManaged()}, introduced to allow deserialization of anonymous inner class */
     @SuppressWarnings("unused") @Deprecated
     private static <T> Predicate<Entity> managedOld() {
```

Comment:
Can we remove this one? Don't think it is use for rebind

Reply:
As above: playing it safe in case someone has extremely old persisted state (until we have a nicer way to walk users though detecting/fixing such problems).